assignments3.txt

```diff
@@ -0,0 +1,8 @@
+1. 
+	a) O(n)
+	
+	b) O(2(n^2))
```

Comment:
Close - you don't need the 2 since big O notation ignores coefficients since their importance decreases as n increases. (if n is equal to 10, 2 * 10 makes a difference, but if n = 1,000,000,000,000, that number vs 2,000,000,000,000 doesn't matter all that much)